### PR TITLE
feat: add arbitrum

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Part of https://github.com/cowprotocol/cowswap/issues/4310

Add Arbitrum

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,11 +27,11 @@
     "prepare": "npm run build",
     "prepublishOnly": "npm test && npm run lint",
     "graphql:codegen": "graphql-codegen --config graphql-codegen.yml",
-    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/v2.252.0/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false",
+    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/varbitrum/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false",
```
Comment (anxolin) [inline package.json:30]: we will need to remember to come back here, right?

File: src/order-book/api.ts [hunk 2]
```diff
@@ -47,6 +48,7 @@ export const ORDER_BOOK_PROD_CONFIG: ApiBaseUrls = {
 export const ORDER_BOOK_STAGING_CONFIG: ApiBaseUrls = {
   [SupportedChainId.MAINNET]: 'https://barn.api.cow.fi/mainnet',
   [SupportedChainId.GNOSIS_CHAIN]: 'https://barn.api.cow.fi/xdai',
```
Comment (anxolin) [inline src/order-book/api.ts:50]: weren't these names changing?

**Review by anxolin — APPROVED** [review]

File: src/order-book/api.ts (hunk 2 above)
Comment (alfetopito) [reply src/order-book/api.ts:50]: No changes yet afaict, just the proposal for a standardization.

File: README.md [hunk 3]
```diff
@@ -127,19 +130,22 @@ import { OrderBookApi } from '@cowprotocol/cow-sdk'
 
 const orderBookApi = new OrderBookApi({
   chainId: SupportedChainId.GNOSIS_CHAIN,
-  baseUrls: { // <-----
+  baseUrls: {
+    // <-----
     [SupportedChainId.MAINNET]: 'https://YOUR_ENDPOINT/mainnet',
-    [SupportedChainId.GNOSIS_CHAIN]: 'https://YOUR_ENDPOINT/gnosis-chain',
+    [SupportedChainId.GNOSIS_CHAIN]: 'https://YOUR_ENDPOINT/gnosis_chain',
```
Comment (mfw78) [inline README.md:136]: :heart_eyes:

Comment (alfetopito) [reply README.md:136]: As requested, your Highness.

File: package.json (hunk 1 above)
Comment (anxolin) [reply package.json:30]: but i thought we do tagged versions. But anyways, fine, next version we change it

**Pull request merged**